tests/DFE.FindInformationAcademiesTrusts.CypressTests/cypress/pages/trusts/ofstedPage.ts

```diff
@@ -1,6 +1,8 @@
 import { TableUtility } from "../tableUtility";
 
 class OfstedPage {
+    dateRegex = /^\d{1,2} (Jan|Feb|Mar|Apr|May|Jun|Jul|Aug|Sept|Oct|Nov|Dec) \d{4}$|^No data$/;
```

Comment:
Could you just put a comment above this regex that gives an indication of what it resolves